src/services/meta_transaction_service.ts

```diff
@@ -216,7 +217,7 @@ export class MetaTransactionService implements IMetaTransactionService {
             endpoint,
             isUnwrap: false,
             isWrap: false,
-            metaTransactionVersion: 'v1',
+            metaTransactionVersion: params.metaTransactionVersion,
```

Comment:
any updated needed in `meta_transaction_test.ts`?

Reply:
Nothing related to this for now 👍 `params.metaTransactionVerion` can only be `v1` for now since meta-txn v2 is not live yet. Will need to add more tests when we add support for meta-txn v2 but good for now 🚀 I also added some tests in `meta_transaction_test.ts` but not related to this